apps/accounts/forms/update.py

```diff
@@ -129,3 +133,30 @@ def save(self, user, *args, **kwargs):
             }
             rollbar.report_exc_info(request=request, extra_data=extra_data)
         return user
+
+
+class TimeZoneForm(AbstractModelInstanceUpdateForm):
+    class Meta:
+        model = UserProfile
+        fields = (
+            'timezone',
+        )
+
+    def __init__(self, instance, *args, **kwargs):
+        """Initialization for TimeZoneForm.
+        """
+        user = instance
+        user_profile = user.profile
```

Comment:
Just pass in `user_profile` in the constructor

```
user_profile = instance
```

Reply:
Done